data/json/itemgroups/Locations_MapExtras/locations_waterbody.json

```
@@ -0,0 +1,24 @@
+[
+  {
+    "id": "crate_ship_cargo",
+    "type": "item_group",
+    "subtype": "distribution",
+    "items": [ { "group": "military", "prob": 100 }, { "group": "rare", "prob": 100 }, { "group": "science", "prob": 100 } ]
+  },
+  {
+    "id": "lake_bed",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [ { "group": "fishing_items_trash", "prob": 6 } ]
+  },
+  {
+    "id": "underwater_toxic_waste",
+    "type": "item_group",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "toxic_waste", "prob": 200 },
+      { "group": "toxic_dump_equipment", "prob": 20 },
+      { "item": "mutagen_fish", "prob": 20 }
```

Note to self: in a followup PR, add back a chance for `toxic_waste` to contain large barrels of generic mutagen because it's always funny when the player finds those.
```suggestion
      { "group": "toxic_dump_equipment", "prob": 20 }
```